pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -739,22 +757,40 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 		return
 	}
 
-	// Step 5: start the init container.
-	if container := podContainerChanges.NextInitContainerToStart; container != nil {
-		// Start the next init container.
+	// Helper containing boilerplate common to starting all types of containers.
+	// typeName is a label used to describe this type of container in log messages.
```

Comment:
nit: list the types that are currently in use.

Reply:
Done.